From f4e6ecd41007b5bb1f1fe99af422b7208f8d1643 Mon Sep 17 00:00:00 2001
From: Cedric Poignet <cedric.poignet@intel.com>
Date: Wed, 27 Mar 2013 12:12:13 +0100
Subject: [PORT FROM MAIN][BT] Use HID USB bus to avoid BlueZ Kernel dependency

BZ: 96190

Using BUS_USB instead of BUS_BLUETOOTH allows us to get rid
of the driver registration for the BUS_BLUETOOTH made in the
HIDP kernel layer.

Change-Id: Ie0e1e22af1b41268655710bdcde73917f38db0dd
Orig-Change-Id: Ifd1fbc4b60fd857c9bb1859287aedb65ad8ea699
Category: device enablement
Domain: CWS.BT-Common
Origin: internal
Upstream-Candidate: yes
Signed-off-by: Cedric Poignet <cedric.poignet@intel.com>
Reviewed-on: http://android.intel.com:8080/99669
Reviewed-by: Bondier, CedricX <cedricx.bondier@intel.com>
Tested-by: Champciaux, Nicolas <nicolas.champciaux@intel.com>
---
 btif/co/bta_hh_co.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/btif/co/bta_hh_co.c b/btif/co/bta_hh_co.c
index 51edcd4..ee634d3 100644
--- a/btif/co/bta_hh_co.c
+++ b/btif/co/bta_hh_co.c
@@ -421,7 +421,7 @@ void bta_hh_co_send_hid_info(btif_hh_device_t *p_dev, char *dev_name, UINT16 ven
     strncpy((char*)ev.u.create.name, dev_name, sizeof(ev.u.create.name) - 1);
     ev.u.create.rd_size = dscp_len;
     ev.u.create.rd_data = p_dscp;
-    ev.u.create.bus = BUS_BLUETOOTH;
+    ev.u.create.bus = BUS_USB;
     ev.u.create.vendor = vendor_id;
     ev.u.create.product = product_id;
     ev.u.create.version = version;
-- 
1.7.4.1

